src/.vscode/launch.json

```diff
@@ -0,0 +1,28 @@
+{
```

Comment:
VS Code is not required to run the sample so I would keep it minimal and not include this

Reply:
These are under src/ and are actually used to build the production code. They are relocated from the root of the repo.
The reason behind is to reduce the interference to the prod code from the examples.

I would not say this is the best place to put this change but at the same time, I don't think it worth another PR just for the relocate of these jsons.